go.mod

```diff
@@ -255,7 +253,7 @@ require (
 )
 
 // Using a fork of Prometheus with Mimir-specific changes.
-replace github.com/prometheus/prometheus => github.com/grafana/mimir-prometheus v0.0.0-20231207110551-060dc59065d8
+replace github.com/prometheus/prometheus => github.com/grafana/mimir-prometheus v0.0.0-20231211122056-f51e8aae4265
```

Comment:
I'm a little confused about this; I would've expected this to already be in `main`.

Reply:
Hmm...maybe the one already in `main` is actually more current despite the timestamp being later? I'm going to close this and start again fresh.